src/stores/destinations.json

```
@@ -179,6 +179,20 @@
     "createdAt": "2020-02-26T09:17:46.291Z",
     "updatedAt": "2020-10-30T09:35:23.487Z"
   },
+  {
+    "config": {
+      "destConfig": { "defaultConfig": ["pushKey"] },
+      "secretKeys": ["pushKey"],
+      "excludeKeys": [],
+      "includeKeys": ["pushKey"],
```

The `includeKeys` is needed for `device-mode` destinations. As we're doing only the cloud-mode, we won't need this.
```suggestion
      "includeKeys": [],
```

---

✅